packages/@react-spectrum/dropzone/src/DropZone.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {AriaLabelingProps, DOMProps, FocusableRefValue, StyleProps} from '@react-types/shared';
+import {classNames, createFocusableRef, SlotProvider, useStyleProps} from '@react-spectrum/utils';
+import {DropZoneProps, FileTriggerContext, Provider, DropZone as RACDropZone} from 'react-aria-components';
+import {mergeProps} from '@react-aria/utils';
+import React, {ReactNode, Ref, useImperativeHandle, useRef} from 'react';
+import styles from '@adobe/spectrum-css-temp/components/dropzone/vars.css';
+export interface SpectrumDropZoneProps extends DropZoneProps, DOMProps, StyleProps, AriaLabelingProps {
+  children: ReactNode,
+  isFilled?: boolean, 
+  bannerMessage?: string
+}
+
+export interface DropZoneRef extends FocusableRefValue<HTMLInputElement, HTMLDivElement> {
+  getInputElement(): HTMLInputElement | HTMLDivElement | null
+}
+
+function DropZone(props: SpectrumDropZoneProps, ref: Ref<DropZoneRef>) {
+  let {children, isFilled, bannerMessage, ...otherProps} = props;
+  let {styleProps} = useStyleProps(props);
+  let domRef = useRef<HTMLDivElement>(null);
+  let inputRef = useRef<HTMLInputElement>(null);
+
+  // Expose imperative interface for ref
```

Comment:
leaving imperative handle here for now. we can decide to remove it later but just gonna leave these questions here for us to think about for the future:
- Do we think that RAC components should have imperative refs? 
- Does the FileTriggers input store any important information that we’d like the user to be able to access or have any interactions that the user may want to trigger?